README.md

```diff
@@ -40,6 +41,7 @@ database/sql does not allow to use big uint64 values.
 It is recommended use type `UInt64` which is provided by driver for such kind of values.
 type `[]byte` are used as raw string (without quoting)
 for passing value of type `[]uint8` to driver as array - please use the wrapper `clickhouse.Array`
+for passing decimal value please use the wrappers `clickhouse.Decimal*`
```

Comment:
Why isn't there a type wrapper `clickhouse.DecimalX(X, Y)`?

Reply:
because clickhouse have functions like `toDecimalX(value, S)`, that means clickhouse.Decimal should accept 3 arguments, X, value and S. 
IMHO it would be better to follow clickhouse functions signature.